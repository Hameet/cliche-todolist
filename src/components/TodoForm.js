import React from 'react'
import styled from 'styled-components'

const StyledInput = styled.input`
  color: paleviolet;
  width: 200px;
  border-radius: 5px;
  border-color: red;
`;

StyledInput.defaultName = 'StyledInput'

export default function TodoForm ({handleBlur}) {
    console.log("hey", {handleBlur})
    return (
        <form>
            <div>
                <label>To do</label>
                <br/>
                <StyledInput name='todo' onBlur ={handleBlur} />
            </div>
        </form>
    )
}