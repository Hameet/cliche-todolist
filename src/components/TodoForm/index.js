import React from 'react'
import styled from 'styled-components'

const StyledInput = styled.input`
  color: paleviolet;
  width: 400px;
  height: 30px;
  border-radius: 5px;
  border-color: red;
`

StyledInput.defaultName = 'StyledInput'

export default function TodoForm ({ handleBlur }) {
  return (
    <form>
      <div>
        <StyledInput name='todo' onBlur={handleBlur} />
      </div>
    </form>
  )
}
