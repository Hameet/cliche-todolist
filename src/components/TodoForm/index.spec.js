import React from 'react'
import { shallow } from 'enzyme'

import TodoForm from '.'


describe('components:TodoForm', function () {
  it('renders the TodoForm and CSS properties properly', () =>
    expect(toJson(shallow(<TodoForm>TodoForm</TodoForm>))
    ).toMatchSnapshot())
})


