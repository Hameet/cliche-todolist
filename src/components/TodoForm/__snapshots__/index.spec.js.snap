// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:TodoForm renders the TodoForm and CSS properties properly 1`] = `
<form>
  <div>
    <label>
      To do
    </label>
    <br />
    <styled.input
      name="todo"
    />
  </div>
</form>
`;
